ace/tests/test_ace.py

```diff
@@ -71,12 +70,13 @@ def test_plos_source(test_data_path, source_manager):
 @pytest.mark.vcr(record_mode="once")
 def test_database_processing_stream(db, test_data_path):
     ingest.add_articles(db, test_data_path + '*.html')
-    assert len(db.articles) == 4  # cannot find pmid for some articles
+    assert len(db.articles) == 5  # cannot find pmid for some articles
```

Comment:
Is this 5 instead of 4 now because you added a springer article?

Reply:
short answer: yes

Wish I would just use all the articles in the folder, but the current code cannot find the pmids in the HTML (I think in some instances it isn't in the HTML), and since the files are not saved with the pmid as the filename, there isn't a straightforward way to identify the pmid and save the article in the database.